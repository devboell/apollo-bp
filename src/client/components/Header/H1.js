import styled from 'styled-components'

const H1 = styled.h1`
  margin: 5px 0px 0px 50px;
`

export default H1
