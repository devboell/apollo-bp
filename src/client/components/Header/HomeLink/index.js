import React from 'react'

import { Link } from 'react-router-dom'
import homeImage from './images/home.png'
import Image from './Image'

const HomeLink = () =>
  <Link to="/" >
    <Image src={homeImage} alt="Home" />
  </Link>

export default HomeLink
