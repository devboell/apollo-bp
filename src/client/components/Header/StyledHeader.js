import styled from 'styled-components'

const StyledHeader = styled.header`
  display: flex;
  vertical-align: center;
  padding: 10px 0px 20px 30px;
`

export default StyledHeader
