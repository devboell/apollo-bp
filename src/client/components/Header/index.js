import React from 'react'

import StyledHeader from './StyledHeader'
import HomeLink from './HomeLink'
import H1 from './H1'

const Header = () =>
  <StyledHeader>
    <HomeLink />
    <H1>Ukiyo-e</H1>
  </StyledHeader>

export default Header
