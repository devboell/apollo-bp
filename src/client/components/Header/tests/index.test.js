import React from 'react'
import Header from '../index'

it('Header component snapshot', () => {
  const wrapper = shallow(
    <Header />,
  )
  expect(wrapper).toMatchSnapshot()
})
