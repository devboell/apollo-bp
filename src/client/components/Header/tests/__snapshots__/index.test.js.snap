// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component snapshot 1`] = `
<styled.header>
  <HomeLink />
  <styled.h1>
    Ukiyo-e
  </styled.h1>
</styled.header>
`;
